driver/monitoring/prometheus/prometheus_test.go

```diff
@@ -19,12 +19,13 @@ package prometheusmon
 import (
 	"bytes"
 	"fmt"
-	"github.com/Fantom-foundation/Norma/driver/network"
 	"io"
 	"net/http"
 	"testing"
 	"time"
 
+	"github.com/Fantom-foundation/Norma/driver/network"
+
```

Comment:
remove this empty line?

Reply:
Not sure why these aren't caught automatically by gofmt/goimports.
Maybe some missing flags?